website/docs/r/iploadbalancing_http_frontend.markdown

```diff
@@ -50,7 +50,7 @@ The following arguments are supported:
 * `default_ssl_id` - Default ssl served to your customer
 * `disabled` - Disable your frontend. Default: 'false'
 * `ssl` - SSL deciphering. Default: 'false'
-
+* `redirect_location` - Redirection HTTP'
```

Comment:
usually, i try to "copy/paste" the official api field description
( but im not always consistent on this)

"HTTP redirection (Ex : http://www.ovh.com)"